src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -544,24 +583,27 @@ object IEnumerator.Current
 
             public bool MoveNext()
             {
-                if (_version == _builder._version)
+                ReadOnlyCollectionBuilder<T> localBuilder = _builder;
```

Comment:
> I see that issue have up for grab label. So I do not understand correctly this issue

I see.  @karelz, @VSadov, we should clarify what the up-for-grabs label means, and if the intent is that it means we've agreed that the issue should be fixed in entirety, make sure we've agreed on the behavioral changes described in the issue.